### PR TITLE
Operator modals

### PR DESCRIPTION
**Pull request by paul4702**

### CODE REVIEW

File: src/lib/components/v2/inputs/TextInputWithEndButton.svelte [hunk 1]
```diff
@@ -4,21 +4,70 @@
 	import { cn } from '$lib/utils/helpers/commonHelper';
 
 	export let styleClass = '';
-	export let tooltipText = '';
+	// export let tooltipText = '';
```
Comment (vishwajeetraj11) [inline src/lib/components/v2/inputs/TextInputWithEndButton.svelte:7]: please remove this comment if not needed.

File: src/lib/page-components/v2/oyster/operator/Modals/RegisterModal.svelte [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+			default:
+				break;
+		}
+	}
+</script>
+
+<Modal modalFor="oyster-register-url-operator">
+	<svelte:fragment slot="title">Register</svelte:fragment>
+	<svelte:fragment slot="content">
+		<!-- <p class="pb-3 text-base font-normal">Details</p> -->
```
Comment (vishwajeetraj11) [inline src/lib/page-components/v2/oyster/operator/Modals/RegisterModal.svelte:66]: please remove this comment if not needed.

**Review by vishwajeetraj11 — APPROVED** [review]

File: src/lib/page-components/v2/oyster/operator/Modals/RegisterModal.svelte [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+	export let registered: boolean;
+
+	export let handleOnUpdate = () => {};
+	export let updateLoading = false;
+	export let registeredCpURL = '';
+
+	let isUpdateButtonDisabled: boolean;
+	let isRegistredButtonDisabled: boolean;
+
+	$: isUpdateButtonDisabled =
```
Comment (souvikmishra) [inline src/lib/page-components/v2/oyster/operator/Modals/RegisterModal.svelte:23]: lets keep the reactive statements at the end only, also beware that although all the reactive statements execute at the end (i.e. when the DOM is painted), the order in which the are written (wrt to reactive statements) matter at the the time of execution

File: src/lib/page-components/v2/oyster/operator/Modals/RegisterModal.svelte [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+		!validCPUrl || registeredCpURL === updatedCpURL || !enableRegisterButton;
+	$: isRegistredButtonDisabled = !validCPUrl || !enableRegisterButton;
+
+	console.log({ isUpdateButtonDisabled });
```
Comment (souvikmishra) [inline src/lib/page-components/v2/oyster/operator/Modals/RegisterModal.svelte:27]: maybe remove the console log if not needed

File: src/lib/page-components/v2/oyster/operator/OysterOperatorRegisterPage.svelte [hunk 5]
```diff
@@ -228,88 +257,94 @@
...
+						Register
+					</ModalButton>
 				{/if}
-			{:else}
-				<ConnectWalletButton isLarge={true} />
 			{/if}
 		</div>
+		{#if registered}
+			<div class="mt-6 rounded-3xl bg-white px-8 py-6">
+				<!-- <p class="pb-3 text-base font-normal">Control Plane:</p> -->
```
Comment (souvikmishra) [inline src/lib/page-components/v2/oyster/operator/OysterOperatorRegisterPage.svelte:291]: maybe remove the comments if not needed

**Review by souvikmishra — APPROVED** [review]

**Pull request merged**